WORKSPACE.bzlmod

```diff
@@ -0,0 +1 @@
+# Intentionally left blank as all deps are defined in MODULE.bazel.
```

Comment:
For my own understanding: Why have this file if we keep it empty? Is it required by the module conventions?

Reply:
I added a comment explaining this, it's a rather technical point.